jacoco-config/enable-jacoco

```diff
@@ -0,0 +1,86 @@
+package hudson.plugins.jacoco;
+
+import com.cloudbees.hudson.plugins.folder.*
+import hudson.plugins.jacoco.report.CoverageReport
+
+/*Loop through all items inside a folder*/
+void processFolder(Item folder) {
+    folder.getItems().each{
```

Comment:
Any possibility of the result of folder.getItems() being NULL? If it is possible this could cause an NPE.

Reply:
Since, there has to be a job for this configuration to be applied for it is highly unlikely that it would cause a NPE.